bin/pygrb/pycbc_pygrb_efficiency

```diff
@@ -428,48 +417,48 @@ if do_injections:
     # NOTE: the loop on num_mc_injs would fill up the *_inj['dist_mc']'s at the
     # same time, so filling them up sequentially will vary the numbers a little
     # (this is an MC, order of operations matters!)
-    found_inj['dist_mc'] = ppu.mc_cal_wf_errs(num_mc_injs, found_inj['dist'],
+    found_inj_dist_mc = ppu.mc_cal_wf_errs(num_mc_injs, injs['found/distance'],
```

Comment:
Is there a reason for using `found_inj_dist_mc` rather than ` found_inj['dist_mc']` (cleaner)?  A similar question applies to `missed_inj_dist_mc`